### PR TITLE
Move some Support files to GTClang

### PR DESCRIPTION
**Pull request by jdahm**
## Technical Description

Moves some files in dawn/Support to gtclang/Support and remove the now-unused Twine.

### CODE REVIEW

**Comment by jdahm** [comment]
launch jenkins

**Comment by jdahm** [comment]
Presumably more could now be removed since these are in GTClang itself (that links to LLVM and Clang).

File: dawn/src/dawn/Support/StringUtil.h [hunk 1]
```diff
@@ -129,4 +129,16 @@ extern std::string decimalToOrdinal(int dec);
 
 /// @}
 
+/// @brief Checks if two strings are equal (case insensitive)
+/// @ingroup support
+bool equals_lower(const std::string& a, const std::string& b);
```
Comment (mroethlin) [inline dawn/src/dawn/Support/StringUtil.h:134]: these should probably be camelCase for consistency w/ rest of codebase

Comment (jdahm) [reply dawn/src/dawn/Support/StringUtil.h:134]: I thought that we wanted standalone methods to use snake case. @twicki?

Comment (jdahm) [reply dawn/src/dawn/Support/StringUtil.h]: @twicki and I discussed this and landed on camelCase, so I changed it to use that.

**Comment by jdahm** [comment]
launch jenkins

**Pull request merged**